validation/src/test/scala/hmda/validation/rules/lar/validity/V460Spec.scala

```diff
@@ -0,0 +1,27 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.validation.dsl.{ Failure, Success }
+import hmda.validation.rules.lar.LarEditCheckSpec
+import org.scalacheck.Gen
+
+class V460Spec extends LarEditCheckSpec {
+  property("CoApplicant ethnicity must = 1,2,3,4 or 5") {
+    forAll(larGen) { lar =>
+      V460(lar) mustBe Success()
+    }
+  }
+
+  val badCoEthnicityGen: Gen[Int] = {
+    val belowRange = Gen.choose(Integer.MIN_VALUE, 0)
+    val aboveRange = Gen.choose(6, Integer.MAX_VALUE)
+    Gen.oneOf(belowRange, aboveRange)
+  }
```

Comment:
I vote for extracting this function into some common place, perhaps `LarGenerators` or similar.  (It's not LAR-specific, but I don't know of a better place.  You could create one if so inspired, but I'd be fine with just putting it into the pre-existing place.)

On second thought, you know what?  I could just merge this PR and make that change myself.  Less WIP, that way.